### DISCUSSION TITLE
Coloring of Start/Stop button

### DISCUSSION THREAD

**Discussion by powerjungle** [vmiklos/plees-tracker#387] [Ideas]
Currently on my phone, the Start/Stop button remains grey no matter what. It would be nice if for example the button is green when the text is "Start" and red when it's "Stop". It's a very small change, but it might make the app a bit easier to follow.

**Comment by vmiklos** [comment]
Do you use dark mode? Light mode sets the start button to purple, the stop button to magenta.

**Reply by powerjungle** [reply]
Yeah I'm using dark mode, forgot to mention, sorry. Would be nice if dark mode also has 2 different colors for the different states.

**Reply by vmiklos** [reply]
Care to suggest what RGB colors to use for the start and stop cases in dark mode? :)

**Reply by powerjungle** [reply]
It's hard to say without seeing it on the app, but my wild guess would be #613583 for Start and #463A31 for Stop.

Also, thanks for working on this app, I know it's a hassle, but it's really useful! If I knew Kotlin and Android I would've tried to make a pull request, but sadly I don't.

**Reply by vmiklos** [reply]
Done, see <https://github.com/vmiklos/plees-tracker/pull/396>, will land in the next release.

**Reply by powerjungle** [reply]
Nice, thank you!
